session_analyzer.py

```diff
@@ -13,6 +14,7 @@ def __init__(self):
     @asyncio.coroutine
     def analyze(self):
         while True:
+            time.sleep(1)
```

Comment:
I use Python 3.5.  I read this docs. This sample works, but when I use send() to pass the value to the generator, it fails.